Bio/KEGG/Compound/__init__.py

```diff
@@ -153,7 +153,7 @@ def parse(handle):
             record.pathway.append(pathway)
         elif keyword == "FORMULA     ":
             record.formula = data
-        elif keyword == "MASS        ":
```

Comment:
In order to support old and new style files, can you make this:

```python
elif keyword in ("MASS ", "EXACT_MASS "):

Reply:
@peterjc Fixed. Thanks!